src/api/gnosisProtocol/api.ts

```diff
@@ -152,7 +152,18 @@ export async function getQuote(params: FeeQuoteParams): Promise<OrderQuoteRespon
 }
 
 export async function getOrder(chainId: ChainId, orderId: string, env?: CowEnv): Promise<EnrichedOrder | null> {
-  return orderBookApi.getOrder(orderId, { chainId, env })
+  const contextOverride = {
+    chainId,
+    // To avoid passing `undefined` and unintentionally setting the `env` to `barn`
+    // we check if the `env` is `undefined` and if it is we don't include it in the contextOverride
+    ...(env
+      ? {
+          env,
+        }
+      : undefined),
+  }
+
+  return orderBookApi.getOrder(orderId, contextOverride)
```

Comment:
See the relevant SDK code: https://github.com/cowprotocol/cow-sdk/blob/5688fee76a4bdd6f68bbd96691c10f302c451685/src/order-book/api.ts#L164

Reply:
We should fix in the SDK too, should default to prod